drag_and_drop_v2/utils.py

```diff
@@ -37,7 +31,7 @@ def _clean_data(data):
     return cleaned_text
 
 
-ALLOWED_TAGS = bleach.ALLOWED_TAGS + [
+ALLOWED_TAGS = bleach.ALLOWED_TAGS | {
```

Comment:
Idea: this bleach `5.0.1` -> `6.0.0` upgrade does not need to break anything. I suggested an approach in which we could preserve compatibility with Nutmeg and Olive with only five characters of code overhead. We could also unpin the bleach constraint. If we went ahead with this approach, we could deploy it to the `edx-platform` right away as the `3.1.0` version. What do you think?

The Maple support removal looks excellent, of course. Thanks for doing it!

```suggestion
# Convert `bleach.ALLOWED_TAGS` to a set because it is a list in `bleach<6.0.0`.
ALLOWED_TAGS = set(bleach.ALLOWED_TAGS) | {
```